### PR TITLE
new icon: kaggle (original, original-wordmark)

### PR DESCRIPTION
**Pull request by kimjiwook0129**
new icon: kaggle (original, original-wordmark)

- devicon.json file has been edit (the was a huge diff in the previous PR due to my editor's auto-format on save... sorry about that)
- the images work perfectly fine on my environment (opened with chrome). Could you please tell me the environment | browser you're using to open up the images? Let me try to figure out if possible :)
Screen Shot 2021-09-06 at 12 59 57 PM
Screen Shot 2021-09-06 at 12 59 49 PM

### CODE REVIEW

**Review by amacado — COMMENTED** [review]
GitHub seems to have a problem displaying the graphics. When I export them they are working fine, not yet sure about that issue. As you can see in the icomoon screenshot the icon looks broken. It's not centered and way to small.

image

**Comment by kimjiwook0129** [comment]
> GitHub seems to have a problem displaying the graphics. When I export them they are working fine, not yet sure about that issue. As you can see in the icomoon screenshot the icon looks broken. It's not centered and way to small.
> 
> image

I see what the problem is, I'll get to it around tomorrow when I'm available!

**Comment by kimjiwook0129** [comment]
Screen Shot 2021-09-07 at 12 04 02 AM

Just realized, not only mine but also all other svgs in devicon/icons are not rendered properly and dislplayed as a single line.

**Comment by kimjiwook0129** [comment]
Reported as an issue.

File: icons/kaggle/kaggle-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....6-4.28-14.354 0-5.468 1.367-10.104 4.1-13.9 3.2-4.576 7.757-6.865 13.642-6.865 4.458 0 8.202 1.16 11.234 3.478v-1.783c0-0.594 0.296-0.892 0.892-0.892h6.152c0.593 0 0.9 0.298 0.9 0.892v39.854c0 6.476-1.62 11.395-4.858 14.753m-3.076-42.84c-1.487-3.803-4.935-5.706-10.343-5.706-6.955 0-10.432 4.34-10.432 13.017 0 4.814 1.16 8.323 3.477 10.52 1.724 1.725 3.953 2.586 6.687 2.586 5.586 0 9.124-1.902 10.6-5.706v-14.7zm21.575 27.36h-6.153c-0.594 0-0.9-0.296-0.9-0.9v-58.923c0-0.593 0.297-0.9 0.9-0.9h6.153c0.593 0 0.9 0.297 0.9 0.9v58.935c0 0.594-0.298 0.9-0.9 0.9m44.843-18.1c0 0.595-0.3 0.892-0.892 0.892h-28.977c0.356 2.913 1.605 5.35 3.745 7.3 2.437 2.082 5.498 3.12 9.184 3.12 3.268 0 6.2-0.92 8.826-2.764 0.536-0.357 0.98-0.357 1.338 0l4.2 4.28c0.476 0.477 0.476 0.9 0 1.248-4.28 3.2-9.244 4.814-14.9 4.814-6.063 0-11.116-2.02-15.158-6.062-3.922-3.982-5.884-9.064-5.884-15.246 0-5.825 1.93-10.758 5.795-14.8 3.567-3.685 8.203-5.527 13.9-5.527 5.408 0 9.955 1.932 13.642 5.795 3.803 3.983 5.557 8.976 5.26 14.98zm-11.68-13.106c-2.02-1.842-4.458-2.764-7.3-2.764-2.675 0-5.024 0.892-7.043 2.674s-3.3 4.102-3.834 6.955h21.487c-0.06-2.733-1.16-5.023-3.3-6.865" clip-path="url(#b)" fill="#20beff"/></svg>
```
Comment (Panquesito7) [inline icons/kaggle/kaggle-original-wordmark.svg:1]:
```suggestion
<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path transform="matrix(.40593 0 0 .40593 -11.403 -3.7815)" d="m105.75 102.97c-0.06 0.238-0.298 0.357-0.713 0.357h-7.937c-0.477 0-0.9-0.208-1.248-0.625l-13.106-16.672-3.655 3.477v12.93c0 0.595-0.298 0.892-0.892 0.892h-6.152c-0.595 0-0.892-0.297-0.892-0.892v-58.935c0-0.593 0.297-0.9 0.892-0.9h6.153c0.594 0 0.892 0.298 0.892 0.9v36.288l15.692-15.87c0.416-0.415 0.832-0.624 1.248-0.624h8.204c0.356 0 0.593 0.15 0.713 0.445 0.12 0.357 0.1 0.624-0.1 0.803l-16.575 16.046 17.297 21.488c0.237 0.238 0.297 0.535 0.18 0.892m32.144 0.356h-6.152c-0.654 0-0.98-0.296-0.98-0.9v-1.25c-2.616 1.96-5.974 2.942-10.075 2.942-3.744 0-6.985-1.16-9.72-3.476-2.675-2.377-4.012-5.4-4.012-9.094 0-5.825 3.625-9.778 10.877-11.858 2.853-0.833 7.162-1.545 12.928-2.14 0.18-2.198-0.4-4.07-1.738-5.617s-3.285-2.318-5.84-2.318c-3.33 0-6.717 1.2-10.164 3.566-0.535 0.298-0.922 0.238-1.16-0.178l-3.2-4.547c-0.298-0.357-0.208-0.772 0.267-1.248 4.577-3.15 9.332-4.726 14.266-4.726 3.804 0 7.044 0.892 9.72 2.675 3.923 2.616 5.884 6.806 5.884 12.57v24.697c0 0.595-0.298 0.9-0.9 0.9m-7.134-18.622c-5.826 0.595-9.807 1.368-11.947 2.318-2.913 1.248-4.222 3.03-3.924 5.35 0.178 1.308 0.833 2.363 1.962 3.165s2.437 1.264 3.923 1.382c4.16 0.297 7.5-0.803 9.986-3.3zm44.463 34.093c-3.24 3.357-7.832 5.038-13.775 5.038-3.448 0-6.717-0.862-9.807-2.585-0.655-0.417-1.427-0.967-2.318-1.65s-1.933-1.53-3.12-2.54c-0.417-0.356-0.446-0.802-0.1-1.338l4.2-4.2c0.178-0.18 0.416-0.268 0.713-0.268 0.237 0 0.446 0.1 0.624 0.268 3.328 3.328 6.568 4.993 9.718 4.993 7.2 0 10.788-3.745 10.788-11.234v-4.636c-2.794 2.318-6.508 3.477-11.145 3.477-5.766 0-10.284-2.168-13.552-6.508-2.853-3.803-4.28-8.6-4.28-14.354 0-5.468 1.367-10.104 4.102-13.9 3.2-4.576 7.757-6.865 13.642-6.865 4.458 0 8.203 1.16 11.233 3.478v-1.783c0-0.594 0.297-0.892 0.892-0.892h6.153c0.593 0 0.9 0.298 0.9 0.892v39.854c0 6.476-1.62 11.395-4.86 14.753m-3.077-42.84c-1.486-3.803-4.934-5.706-10.342-5.706-6.954 0-10.43 4.34-10.43 13.017 0 4.814 1.16 8.323 3.477 10.52 1.723 1.725 3.952 2.586 6.687 2.586 5.586 0 9.123-1.902 10.6-5.706zm45.006 42.85c-3.24 3.357-7.832 5.038-13.775 5.038-3.45 0-6.718-0.862-9.808-2.585-0.655-0.417-1.427-0.967-2.318-1.65s-1.932-1.53-3.12-2.54c-0.416-0.356-0.446-0.802-0.1-1.338l4.2-4.2c0.178-0.18 0.416-0.268 0.713-0.268 0.237 0 0.446 0.1 0.624 0.268 3.328 3.328 6.567 4.993 9.718 4.993 7.2 0 10.8-3.745 10.8-11.234v-4.636c-2.795 2.318-6.5 3.477-11.145 3.477-5.766 0-10.284-2.168-13.552-6.508-2.854-3.803-4.28-8.6-4.28-14.354 0-5.468 1.367-10.104 4.1-13.9 3.2-4.576 7.757-6.865 13.642-6.865 4.458 0 8.202 1.16 11.234 3.478v-1.783c0-0.594 0.296-0.892 0.892-0.892h6.152c0.593 0 0.9 0.298 0.9 0.892v39.854c0 6.476-1.62 11.395-4.858 14.753m-3.076-42.84c-1.487-3.803-4.935-5.706-10.343-5.706-6.955 0-10.432 4.34-10.432 13.017 0 4.814 1.16 8.323 3.477 10.52 1.724 1.725 3.953 2.586 6.687 2.586 5.586 0 9.124-1.902 10.6-5.706v-14.7zm21.575 27.36h-6.153c-0.594 0-0.9-0.296-0.9-0.9v-58.923c0-0.593 0.297-0.9 0.9-0.9h6.153c0.593 0 0.9 0.297 0.9 0.9v58.935c0 0.594-0.298 0.9-0.9 0.9m44.843-18.1c0 0.595-0.3 0.892-0.892 0.892h-28.977c0.356 2.913 1.605 5.35 3.745 7.3 2.437 2.082 5.498 3.12 9.184 3.12 3.268 0 6.2-0.92 8.826-2.764 0.536-0.357 0.98-0.357 1.338 0l4.2 4.28c0.476 0.477 0.476 0.9 0 1.248-4.28 3.2-9.244 4.814-14.9 4.814-6.063 0-11.116-2.02-15.158-6.062-3.922-3.982-5.884-9.064-5.884-15.246 0-5.825 1.93-10.758 5.795-14.8 3.567-3.685 8.203-5.527 13.9-5.527 5.408 0 9.955 1.932 13.642 5.795 3.803 3.983 5.557 8.976 5.26 14.98zm-11.68-13.106c-2.02-1.842-4.458-2.764-7.3-2.764-2.675 0-5.024 0.892-7.043 2.674s-3.3 4.102-3.834 6.955h21.487c-0.06-2.733-1.16-5.023-3.3-6.865" clip-path="url(#b)" fill="#20beff"/></svg>

```

File: icons/kaggle/kaggle-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path transform="matrix(.52703 0 0 .52703 -30.632 -22.456)" d="m105.75 102.97c-0.06 0.238-0.298 0.357-0.713 0.357h-7.937c-0.477 0-0.89-0.208-1.248-0.625l-13.106-16.672-3.655 3.477v12.93c0 0.595-0.298 0.892-0.892 0.892h-6.152c-0.595 0-0.892-0.297-0.892-0.892v-58.935c0-0.593 0.297-0.89 0.892-0.89h6.153c0.594 0 0.892 0.298 0.892 0.89v36.288l15.692-15.87c0.416-0.415 0.832-0.624 1.248-0.624h8.204c0.356 0 0.593 0.15 0.713 0.445 0.12 0.357 0.09 0.624-0.09 0.803l-16.585 16.046 17.297 21.488c0.237 0.238 0.297 0.535 0.18 0.892" fill="#20beff"/></svg>
```
Comment (Panquesito7) [inline icons/kaggle/kaggle-original.svg:1]:
```suggestion
<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path transform="matrix(.52703 0 0 .52703 -30.632 -22.456)" d="m105.75 102.97c-0.06 0.238-0.298 0.357-0.713 0.357h-7.937c-0.477 0-0.89-0.208-1.248-0.625l-13.106-16.672-3.655 3.477v12.93c0 0.595-0.298 0.892-0.892 0.892h-6.152c-0.595 0-0.892-0.297-0.892-0.892v-58.935c0-0.593 0.297-0.89 0.892-0.89h6.153c0.594 0 0.892 0.298 0.892 0.89v36.288l15.692-15.87c0.416-0.415 0.832-0.624 1.248-0.624h8.204c0.356 0 0.593 0.15 0.713 0.445 0.12 0.357 0.09 0.624-0.09 0.803l-16.585 16.046 17.297 21.488c0.237 0.238 0.297 0.535 0.18 0.892" fill="#20beff"/></svg>

```

File: devicon.json [hunk 3]
```diff
@@ -2350,6 +2350,26 @@
...
+              "original",
+              "original-wordmark"
+            ],
+          "font": [
+              "original",
+              "original-wordmark"
+            ]
+        },
+        "color": "#20BEFF",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:2371]:
```suggestion
      "aliases": [
          {
              "base": "original",
              "alias": "plain"
          },
          {
              "base": "original-wordmark",
              "alias": "plain-wordmark"
          }
      ]
```

**Comment by kimjiwook0129** [comment]
Suggested changes confirmed.

**Comment by Thomas-Boi** [comment]
The SVG does seem to work properly when viewed as a raw file (border and sizing changed by me):
image

I think this is a GitHub issue then.

File: devicon.json [hunk 4]
```diff
@@ -2350,6 +2350,35 @@
...
+              "original",
+              "original-wordmark"
+            ],
+          "font": [
+              "original",
+              "original-wordmark"
+            ]
+        },
+        "color": "#20BEFF",
+      "aliases": [
```
Comment (Panquesito7) [inline devicon.json:2371]: Wrong indentation.
```suggestion
        "aliases": [
```

**Comment by Thomas-Boi** [comment]
Hi @kimjiwook0129,

Is the `k` and `kaggle` in the two versions supposed to be not centered? If not, can you center them please?

**Review by Panquesito7 — COMMENTED** [review]
I'm not sure if the `original` version should be kept here. The original logo has the `kaggle` text on it, not `k`. I would recommend changing the `original-wordmark` to be the `original` version.

**Comment by Thomas-Boi** [comment]
@Panquesito7 I think the `original` version can stay. It's used as the favicon for the site:
image
image

**Comment by Panquesito7** [comment]
> @Panquesito7 I think the `original` version can stay. It's used as the favicon for the site:
> image
> image

Ahh, I see. Thanks for pointing that out. 😅

**Comment by kimjiwook0129** [comment]
> @Thomas-Boi  Is the `k` and `kaggle` in the two versions supposed to be not centered? If not, can you center them please?

The svgs are now centered!
<div>
Screen Shot 2021-09-07 at 2 30 34 PM
Screen Shot 2021-09-07 at 2 31 21 PM
</div>

**Review by Panquesito7 — APPROVED** [review]
Awesome work, @kimjiwook0129! Thanks for your contribution! 😄👍🎉

**Comment by Thomas-Boi** [comment]
Hi @kimjiwook0129,

The icon looks great 😄! We will merge this PR in after our current release is finished. Please wait for a few days and your icon will be added to the repo.

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**